app/controllers/actions/CheckEmailVerificationFilter.scala

```diff
@@ -44,21 +46,22 @@ class CheckEmailVerificationFilterImpl(mode: Option[Mode],
     if (frontendAppConfig.emailVerificationEnabled && !mode.contains(AmendMode)) {
       request.userAnswers.get(BusinessContactDetailsPage) match {
         case Some(contactDetails) =>
-          emailVerificationService.isEmailVerified(contactDetails.emailAddress, request.userId).map {
+          emailVerificationService.isEmailVerified(contactDetails.emailAddress, request.userId).flatMap {
             case Verified =>
               logger.info("CheckEmailVerificationFilter - Verified")
-              None
+              Future(Option.empty[Result])
```

Comment:
Could use `Future.successful(None)`

Reply:
Same reply as below

Reply:
Not too sure if they'll remove it but yeah wherever you use `Option.empty[T]` it will help the compiler. Especially when you have pattern matches and flatmaps that have `None`s it will help with inferring the type.

Reply:
> Note: we should also stop using Some where possible and use apply of `Option`

Only use `Option.apply` when the value can be null

Reply:
??? I meant `Option(value)` is the same as `Some(value)`

Reply:
About this PR, I think we should keep `None` over `Option.empty`, since there's no need to help type inference in this case. Not super important though, so I approved the PR